youtube_dlc/YoutubeDL.py

```diff
@@ -113,6 +113,53 @@
 if compat_os_name == 'nt':
     import ctypes
 
+# Archive tree
+class ArchiveTree(object):
+    def __init__(self, line):
+        self.left = None
+        self.right = None
+        self.line = line
```

Comment:
You could remove the comment and add this under the class `"""Archive tree"""` or some very short description

Reply:
Done and done.